### PR TITLE
Added some developer tools for altering the game state.

### PR DESCRIPTION
**Pull request by rjelte**

### CODE REVIEW

File: src/bitwise/core.cljs [hunk 1]
```diff
@@ -47,14 +47,16 @@
                           :grid-template-rows "30px"
                           :align-items "center"})
 
-(def action-button-styles {:width 100
+(def action-button-styles {:min-width 100
                            :height 25
                            :border "1px solid gray"
-                           :display "flex"
+                           :display "inline-block"
```
Comment (Problematic) [inline src/bitwise/core.cljs]: Does this still center the button content?

File: src/bitwise/core.cljs [hunk 2]
```diff
@@ -87,6 +89,7 @@
                      {})}
        [:div (:pid process)]
        [:div (:name program)]
+       ; TODO: Add an "output column here"
```
Comment (Problematic) [inline src/bitwise/core.cljs]: This doesn't seem to be part of this PR

File: src/bitwise/core.cljs [hunk 3]
```diff
@@ -147,7 +150,11 @@
                                  :disabled (= process-slots-available 0)} "<"]
                        [program-info program]])]]]
      [:div
-      [:h3 {:style {:margin-bottom 8}} "debug"]
+      [:h3 {:style {:margin-bottom 8}} "Developer Tools"]
+      [:div 
+        [action-button {:on-click #(swap! game-state assoc-in [:programs] (into #{} (keys program-catalog)))} "Grant All Programs"]
```
Comment (Problematic) [inline src/bitwise/core.cljs]: As a point of interest, you can also use `assoc`, which takes a single keyword instead of a list of keywords. In practice, though, I like `assoc-in` even for single keywords because it's not much more overhead, and it's more flexible if the path changes.

File: src/bitwise/core.cljs [hunk 4]
```diff
@@ -147,7 +150,11 @@
...
+        [action-button {:on-click #(reset! game-state (default-state))} "Reset Default State"]]
```
Comment (Problematic) [inline src/bitwise/core.cljs]: Can you wrap this in a chicken switch? `js/confirm` shows a browser confirmation alert, and returns true or false

Comment (Problematic) [reply src/bitwise/core.cljs]: Er, I should say, it delegates to `window.confirm`

File: src/bitwise/core.cljs (hunk 1 above)
Comment (rjelte) [reply src/bitwise/core.cljs]: Originally I had swapped out for inline-block to get the button layout behavior I was looking for. But I was able to get the same thing by adding "flex" to the parent. 👍

File: src/bitwise/core.cljs (hunk 2 above)
Comment (rjelte) [reply src/bitwise/core.cljs]: ; TODO: Add a TODO to remove this "TODO" in a future PR.
